### ISSUE TITLE
How much the loss should reduce before reasonable sound can be produced?

### ISSUE THREAD

**Issue by Jeevesh8** [NVIDIA/waveglow#198]
How much should the loss reduce before we can generate recognizable sound from inference? My loss is coming around -13.8.. but inference(without denoiser) is producing just all noise. Is it normal behavior ?

**Comment by Selimonder** [comment]
It's possible that you have overfitted your model. Try converging into -4.5,-5.5 range, it should be safe. Also, try using pre-trained models, you should converge better and faster.

**Comment by rafaelvalle** [comment]
Compute the loss on a held out set not seen during training such that you can understand how well the model is generalizing to samples not seen during training.

**Comment by Jeevesh8** [comment]
Actually, my dataset was not in int16 , that was the problem. Thank you for your replies.

**Comment by ashish-roopan** [comment]
can you please explain what do you meant by your  dataset was not in int16 ?

**Comment by Jeevesh8** [comment]
@ashish-roopan  The audio files should have the data type of INT16 rather than Float32 . Audios are discretized and stored. You sample an audio signal at fixed intervals of time and then store the amplitude. You can record audio as a pulse code modulated(PCM16) one, in which 16 bits are used to represent value assigned to one sample. You can use soundfile library to convert between the two. https://github.com/Jeevesh8/Cross-Lingual-Voice-Cloning/blob/4960a1e864a1c233bbc642084750f8d49cec6116/utils.py#L17
